lib/widgets/footer_section.dart

```diff
@@ -0,0 +1,421 @@
+                      child: InkWell(
+                        onTap: () =>
+                            launch('mailto:hackathon@flutterturkiye.org'),
+                        child: Row(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: [
+                            Icon(
+                              Icons.mail,
+                              color: Colors.white,
+                            ),
+                            Padding(
+                              padding: const EdgeInsets.only(left: 10.0),
+                              child: Text(
+                                'hackathon@flutterturkiye.org',
+                                style: TextStyle(
+                                  color: Colors.white,
+                                  fontWeight: FontWeight.bold,
+                                  fontSize: 18,
+                                  fontFamily: 'Montserrat',
+                                ),
+                              ),
+                            ),
+                          ],
+                        ),
+                      ),
+                    ),
+                    Divider(
+                      indent: 100,
+                      endIndent: 100,
+                      color: Colors.white,
+                      thickness: 0.5,
+                    ),
+                    Align(
+                      alignment: Alignment.centerLeft,
+                      child: Padding(
+                        padding: const EdgeInsets.only(
+                          top: 10,
+                          left: 24,
+                        ),
+                        child: Text(
+                          '© 2021 Flutter Türkiye',
+                          style: const TextStyle(
+                            color: Colors.white,
+                            fontSize: 16,
+                          ),
+                        ),
+                      ),
+                    ),
+                    BottomView(),
+                  ],
+                ),
+              ),
+            ],
+          ),
+        ));
+  }
+}
+
+class MediumWidget extends StatelessWidget {
+  const MediumWidget({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+        alignment: Alignment.bottomCenter,
+        child: Container(
+          padding: const EdgeInsets.only(top: 20),
+          height: MediaQuery.of(context).size.height / 2,
+          width: MediaQuery.of(context).size.width,
+          child: Stack(
+            children: [
+              Container(
+                decoration: BoxDecoration(
+                  image: DecorationImage(
+                    image: AssetImage(Assets.footerBackground),
+                    fit: BoxFit.cover,
+                  ),
+                ),
+              ), // background
+              Container(
```

Comment:
Why did we add this container? It doesn't have any height or width. Is it working?

Reply:
This container added for black filter. It has same height with top widget.